packages-native/progress-circle/src/ProgressCircle.tsx

```diff
@@ -31,7 +32,9 @@ export class ProgressCircle extends Component<Props> {
                     strokeCap={this.styles.fill.lineCapRounded ? "round" : "square"}
                 />
                 {validationMessages.length > 0 && (
-                    <Text style={this.styles.validationMessage}>{validationMessages.join("\n")}</Text>
+                    <Text testID={`${this.props.name}$caption`} style={this.styles.validationMessage}>
```

Comment:
This text component is for validation messages, so I think you shouldn't refer to it as "caption".